sci-libs/nfft/nfft-3.5.3.ebuild

```diff
@@ -14,9 +14,14 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="doc openmp"
 
-RDEPEND="sci-libs/fftw:3.0[threads,openmp?]"
+RDEPEND="sci-libs/fftw:3.0=[threads,openmp?]"
 DEPEND="${RDEPEND}"
 
+PATCHES=(
+	"${FILESDIR}/${P}-gcc15.patch"
+	"${FILESDIR}/${P}-rtc.patch"
+)
```

Comment:
Have these patches been sent upstream? If so, can you link the PR here. If not, could you send them upstream?

Reply:
I've sent one patch, and added link.

The other patch is not upstreamable as is, because it just removes a feature, which is not applicable to Gentoo. I have no way to test that feature. I've mentioned this patch in the other PR though.